Query Builder/queries_add.php

```diff
@@ -85,6 +88,12 @@
         $row->addLabel('active', __('Active'));
         $row->addYesNo('active')->isRequired();
 
+    $values['moduleActionName'] = $values['moduleName'].':'.$values['actionName'];
+    $actions = $queryGateway->selectActionListByPerson($gibbon->session->get('gibbonPersonID'));
```

Comment:
What if the current user does not have access to something? It seems that they won't see the action. Most of the times it will be admins doing this, who have most things, but might not have some things (e.g. on the teaching side)